docs/source/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,27 @@
+{% extends "!autosummary/class.rst" %}
+
+{% block methods %}
+{% if methods %}
+   .. HACK -- the point here is that we don't want this to appear in the output, but the autosummary should still generate the pages.
+      .. autosummary::
+         :toctree:
+      {% for item in all_methods %}
+         {%- if not item.startswith('_') or item in ['__call__'] %}
+         {{ name }}.{{ item }}
+         {%- endif -%}
+      {%- endfor %}
+{% endif %}
+{% endblock %}
+
+{% block attributes %}
+{% if attributes %}
+   .. HACK -- the point here is that we don't want this to appear in the output, but the autosummary should still generate the pages.
+      .. autosummary::
+         :toctree:
+      {% for item in all_attributes %}
+         {%- if not item.startswith('_') %}
+         {{ name }}.{{ item }}
+         {%- endif -%}
+      {%- endfor %}
+{% endif %}
+{% endblock %}
```

Comment:
workaround:

```
{% extends "!autosummary/class.rst" %}

{% block methods %}
   {% set methods = methods | select("ne", "__init__") | list %}
   {% if methods %}
   .. rubric:: Methods

   .. autosummary::
   {% for item in methods %}
      ~{{ name }}.{{ item }}
   {%- endfor %}
   {% endif %}

{% endblock %}
```

Reply:
```
{% set methods = methods | select("ne", "__init__") | list %}
```

- `select("ne", "__init__")` removes `__init__` in `methods`.
-  if we didn't put `| list`, `methods` would become a generator (which is truthy)

Reply:
Thank you for your comment. I'm afraid that I'm a little confused as to what should I do. I'd appreciate it if you can directory edit this branch.

Reply:
You can just replace the current template with my template :)

Reply:
Can you apply my suggestion following the instruction here: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes?